<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>775</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MP3 - Player</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="playlistTableView">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>611</width>
      <height>251</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
    </property>
    <property name="dragEnabled">
     <bool>true</bool>
    </property>
    <property name="dragDropOverwriteMode">
     <bool>false</bool>
    </property>
    <property name="dragDropMode">
     <enum>QAbstractItemView::InternalMove</enum>
    </property>
    <property name="defaultDropAction">
     <enum>Qt::MoveAction</enum>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="textElideMode">
     <enum>Qt::ElideRight</enum>
    </property>
    <property name="gridStyle">
     <enum>Qt::SolidLine</enum>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
   </widget>
   <widget class="QLineEdit" name="songSearchLineEdit">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>371</width>
      <height>41</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Song suchen</string>
    </property>
   </widget>
   <widget class="QSlider" name="songSlider">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>380</y>
      <width>431</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="fastForwardButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>440</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Vorspulen</string>
    </property>
    <property name="text">
     <string>vS</string>
    </property>
    <property name="arrowType">
     <enum>Qt::NoArrow</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="rewindButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>440</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Zurückspulen</string>
    </property>
    <property name="text">
     <string>zS</string>
    </property>
   </widget>
   <widget class="QToolButton" name="nextSongButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>440</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Nächster Song</string>
    </property>
    <property name="text">
     <string>nS</string>
    </property>
    <property name="arrowType">
     <enum>Qt::NoArrow</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="lastSongButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>440</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Letzter Song</string>
    </property>
    <property name="text">
     <string>lS</string>
    </property>
    <property name="arrowType">
     <enum>Qt::NoArrow</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="playStopButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>440</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Song wiedergeben</string>
    </property>
    <property name="text">
     <string>play/stop</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="positionLabel">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>410</y>
      <width>58</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="durationLabel">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>410</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="songNameLabel">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>370</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="interpretLabel">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>370</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QToolButton" name="stopButton">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>30</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Wiedergabe stoppen</string>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QSlider" name="soundVolumeSlider">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>220</y>
      <width>20</width>
      <height>191</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="volumeLabel">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>420</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Lautstärke</string>
    </property>
   </widget>
   <widget class="QToolButton" name="randomSongButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>380</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Zufallswiedergabe</string>
    </property>
   </widget>
   <widget class="QToolButton" name="muteButton">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>450</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Stummschalten</string>
    </property>
    <property name="text">
     <string>Stumm</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="loadPlaylistButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Playlist laden</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>775</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuInfo">
    <property name="title">
     <string>Shortcuts</string>
    </property>
    <addaction name="actionPlaylist_laden_Strg_O"/>
    <addaction name="actionZur_ckspulen_Vorspulen_Strg_Strg"/>
    <addaction name="actionLetzter_Song_N_chster_Song_Strg_L_Strg_N"/>
    <addaction name="actionWiedergabe_anhalten_fortsetzen_Leertaste"/>
   </widget>
   <addaction name="menuInfo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionPlaylist_laden_Strg_O">
   <property name="text">
    <string>Playlist laden (Strg + O)</string>
   </property>
  </action>
  <action name="actionZur_ckspulen_Vorspulen_Strg_Strg">
   <property name="text">
    <string>Zurückspulen / Vorspulen (Strg + &lt;- / Strg + -&gt;)</string>
   </property>
  </action>
  <action name="actionLetzter_Song_N_chster_Song_Strg_L_Strg_N">
   <property name="text">
    <string>Letzter Song / Nächster Song (Strg + L / Strg + N)</string>
   </property>
  </action>
  <action name="actionWiedergabe_anhalten_fortsetzen_Leertaste">
   <property name="text">
    <string>Wiedergabe anhalten / fortsetzen (Leertaste)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
